config.rb

```diff
@@ -54,3 +54,10 @@
   # Minify Javascript on build
   # activate :minify_javascript
 end
+
+###
+# Tech Docs-specific configuration
+###
+
+# Host to use for canonical URL generation
```

Comment:
I'd add 'without trailing slash' (or modify the code so that it strips any trailing slash) as I can see this being misconfigured.